app/views/microcosms/show.html.erb

```diff
@@ -0,0 +1,115 @@
+<% content_for :heading do %>
+  <h1><%= "Microcosm" %></h1>
+<% end %>
+
+<style>
+  .mic_row { display: flex }
+  .mic_row > div { padding: 10px; }
+  .mic_details h1 { margin-top: 0; }
+  .mic_details label { font-weight: bold; }
+  .mic_details ul { display: inline-block; }
+  .mic_details ul > li { display: inline-block; }
+  .members > a
+  {
+    display: block;
+    width: 8em;
+    height: 8em;
+    border: 1px solid #888;
+    border-radius: 4px;
+    float: left;
+    margin: 1em;
+    padding: 1em;
+    background-color: rgb(238, 238, 238);
+    box-sizing: border-box;
+    text-decoration: none;
+  }
+</style>
```

Comment:
More inline styles.

Reply:
I was just telling Simon in matrix that this was done for the purpose of a SotM presentation.  I'll remove all inline style.